buildSrc/src/main/java/org/elasticsearch/test/NamingConventionsCheck.java

```diff
@@ -47,10 +44,12 @@
  */
 public class NamingConventionsCheck {
     public static void main(String[] args) throws IOException, ClassNotFoundException {
-        NamingConventionsCheck check = new NamingConventionsCheck();
+        int i = 0;
+        NamingConventionsCheck check = new NamingConventionsCheck(
+                Thread.currentThread().getContextClassLoader().loadClass(args[i++]),
```

Comment:
This seems very wrong...we should not need the context classloader for anything...or even need to get at a classloader? `Class.forName` should work?